help/policies/code_of_conduct.md

```diff
@@ -0,0 +1,69 @@
+
+Failure to comply with this Code of Conduct may result in a warning or suspension from participation in arXiv.
+
+### Stewardship
+
+arXiv is an open access resource. We are dedicated to the permanent custodial preservation of the scholarly record and to the rapid dissemination of scholarly scientific research. We are sustained by a robust community of scientists from around the world who trust us with their work and ideas, and by the financial support of individuals and organizations. 
+
+arXiv looks to the needs of the international scientific community when we set priorities and while we engage in continual development. We are a partner in ongoing open access development and pleased to share our 30 years of experience.
+
+### Ethics and Responsibility
+
+arXiv community members should uphold intellectual honesty and ethics based on community and academic standards when writing and handling scholarly works. We strive to be objective and transparent in our decisions about policies, standards, and practices.
+
+Community members assume responsibility to represent themselves and their work honestly, to be accountable for their own decisions, and to follow arXiv [policies](#policies). Community members should respect the boundary between personal privacy and scholarly openness.
+
+### Engagement
+
+All who participate in the arXiv community contribute to scientific discourse and discovery. arXiv users, volunteers, and staff facilitate scientific communication within a highly dependable service. Daily operations are supported by the researchers who voluntarily share their work and by the volunteers who moderate submissions. As part of arXiv’s continuing growth and evolution, we welcome feedback from the communities we support.
+
+### Inclusiveness and Respect
+
+We encourage and support engagement from a diversity of users in our community. arXiv disseminates scholarly research from members of the communities that we serve without regard to age, creed, disability, ethnicity, race, color, religion, gender, gender identity or expression, marital status, national origin, language, religion, sexual orientation, military or veteran status, or any combination of these or related factors. 
+
+We expect members of the arXiv community to foster a safe environment for communication of diverse views. All communication between arXiv community members should be respectful. 
+
+
+## Code of Conduct Contact
+
+If you have questions or concerns regarding the arXiv CoC, please contact:
+
+- [Oya Rieger](mailto:oyr1@cornell.edu), arXiv Program Director, or
+- [Steinn Sigurdsson](mailto:ss3783@cornell.edu), arXiv Scientific Director, or 
+- [Licia Verde](mailto:liciaverde@icc.ub.edu), Chair of arXiv Scientific Advisory Board, or
+- [Greg Morrisett](mailto:jgm19@cornell.edu), Dean of the Faculty of Computing and Information Science at Cornell University, or
+- [Cornell University Ombudsman](https://www.ombudsman.cornell.edu/)
+
+<span id="policies"></span>
```

Comment:
Intentional?

Reply:
Yes, the markdown converter treats the span like an anchor. The headings turn into anchors in markdown, but headings are subject to change.

Reply:
Sounds good. Forgot about that behavior.